### PR TITLE
Fixes [CascadingCallWrapping] False negative

### PR DESCRIPTION
**Pull request by gouri-panda**
closes: #5042

### CODE REVIEW

File: detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/CascadingCallWrappingSpec.kt [hunk 1]
```diff
@@ -32,6 +32,39 @@ class CascadingCallWrappingSpec {
...
+            fun bar2(): String = ""
+            fun bar3(): String = ""
+            fun zoo() {
+            foo.bar(
+                ""
+            ).bar2()
+        }
+        """
+
+        assertThat(subject.compileAndLint(code)).isNotEmpty
```
Comment (BraisGabin) [inline detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/CascadingCallWrappingSpec.kt:49]: Define the number of findings that you expect, for example, `.hasSize(1)`.

**Comment by BraisGabin** [comment]
> @BraisGabin It's for #5042

Then edit the body of the PR. Otherwise when we merge this the wrong issue will be closed.

**Comment by BraisGabin** [comment]
Hello @gouri-panda. Do you plan to continue with this PR or can we close it so any one can grab the issue and work with it?

**Comment by gouri-panda** [comment]
@BraisGabin Thanks for reminding me! I'll finish this ASAP :)

**Pull request closed without merging**